rewrite-gradle/src/test/java/org/openrewrite/gradle/util/GradleWrapperTest.java

```diff
@@ -35,7 +35,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class GradleWrapperTest {
+class GradleWrapperTest {
```

Comment:
```suggestion
public class GradleWrapperTest {
```

Reply:
Let's see if this is still needed after resolving the conflicts; it it is we can apply this and still see the PR through.